scripts/get_all_heron_paths.sh

```diff
@@ -23,7 +23,19 @@ function query() {
 
 set +e
 # Build everything
-bazel build {heron,integration-test,tools/java}/...
+os_type=darwin
+case "$OSTYPE" in
+  darwin*)
+    os_type=darwin
+    ;;
+  linux*)
+    os_type=ubuntu
```

Comment:
+1 - we should make this as a generic script. @caofangkun - would appreciate if you could detect the OS and include it in get_all_heron_paths()